exonum/src/storage/db.rs

```diff
@@ -23,14 +23,29 @@ use super::Result;
 use self::NextIterValue::*;
 
 /// A set of serial changes that should be applied to a storage atomically.
-pub type Patch = HashMap<String, BTreeMap<Vec<u8>, Change>>;
+#[derive(Debug, Clone)]
+pub struct Patch {
+    pub(crate) changes: HashMap<String, BTreeMap<Vec<u8>, Change>>,
```

Comment:
Shall we make `changes` field crate-private and allow any code to use the underlying map, or add an API to perform operations on a _patch_? In the latter case you will have 

```
self.patch.get_changes(name)  // and
self.patch.contains_changes(name) // (or even a better name of methods)

// instead of

self.patch.changes.get(name)
self.patch.changes.contains_key(name)
```

Reply:
I expect to see here:

```rust
pub fn changes(&self, name: &str) -> Option<Changes>
```

where `Changes` is our type that implement `Iterator` over `Change`.